src/app/models/User.ts

```diff
@@ -137,4 +137,14 @@ export class SessionUser implements SessionUserInterface {
     this.authToken = user.authToken;
     this.userName = user.userName;
   }
+
+  /**
+   * Convert class back to interface for parsing to session cookie
+   */
+  toJson(): SessionUserInterface {
```

Comment:
I'd be cautious doing this. You'll need to serialize the whole model on say the user edit page.

Perhaps add a helper method (or helper prop which returns an object which has `toJson`)

Reply:
I added this because I was concerned that JSON.stringify wasn't working on the class. However I've double check and its not needed.

Reply:
Removed in https://github.com/QutEcoacoustics/workbench-client/commit/db24720085c273257ee40e4c91650e9f9614955a